ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/models/Type.java

```diff
@@ -72,4 +89,19 @@ public String generateMatcher() {
         outputBuilder.append(CHelper.indent(1)).append("}").append(System.lineSeparator());
         return outputBuilder.toString();
     }
+
+    public String getTypeElementsList() throws ParseException {
+        final StringBuilder outputBuilder = new StringBuilder();
+        for ( final Ds3Element element : this.elements) {
+            LOG.debug("element[" + element.getName() + "][" + element.getType() + "]");
+            outputBuilder.append(CHelper.indent(1));
+            outputBuilder.append(CHelper.elementTypeToString(element));
+            outputBuilder.append(" ");
+            outputBuilder.append(Helper.camelToUnderscore(Helper.unqualifiedName(element.getName())));
+            outputBuilder.append(";");
+            outputBuilder.append(System.lineSeparator());
```

Comment:
The functionality is the same, I just wanted to point it out.  Personally, I will just have long lines of code on multiple lines for better layout. Ex:
builder.append("one")
    .append("two")
    .append("three");